src/Traits/RandomValuesTrait.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PNX\DrupalTestUtils\Traits;
+
+use Drupal\Component\Utility\Random;
+use Symfony\Component\Validator\ConstraintViolation;
+
+/**
+ * Additional random value generation methods.
+ */
+trait RandomValuesTrait {
+
+  /**
+   * Generates a random URL.
+   *
+   * @return string
+   *   Random URL.
+   */
+  protected function randomUrl(): string {
+    $random = new Random();
+    return \sprintf('https://%s.com/%s', $random->name(), $random->name());
+  }
```

Comment:
I've always felt we should use `example.com` as the domain name. Turns out there is now a reserved [.example](https://en.wikipedia.org/wiki/.example) TLD.
```suggestion
  /**
   * Generates a random URL.
   *
   * @return string
   *   Random URL.
   */
  protected function randomUrl(): string {
    $random = new Random();
    return \sprintf('https://%s.example/%s', $random->name(), $random->name());
  }
```